docs/examples/index.json

```diff
@@ -351,6 +351,14 @@
         "filePath": "201/web-app-loganalytics/main.bicep",
         "description": "201/web-app-loganalytics"
     },
+    {
+        "filePath": "201/web-app-loganalytics-mod/main.bicep",
+        "description": "201/web-app-loganalytics-mod"
+    },
+    {
+        "filePath": "201/web-app-conditional-log/main.bicep",
```

Comment:
this needs to be removed I think

Reply:
....yup didn't realize I left in there as well.